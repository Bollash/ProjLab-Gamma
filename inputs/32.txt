Create Space
Mod Counter 2
Countdown
Status All