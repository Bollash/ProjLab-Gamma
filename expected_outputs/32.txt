Space létrehozva
A napszél visszaszámláló a kapott értékre állítódott.
A napszél számláló eggyel csökkent.
Space
0
1
10
False
0
0